pkg/gardenlet/controller/seed/seed/components.go

```diff
@@ -125,6 +125,7 @@ func defaultIstio(
 		Namespace:             *conf.SNI.Ingress.Namespace,
 		ProxyProtocolEnabled:  gardenletfeatures.FeatureGate.Enabled(features.APIServerSNI),
 		VPNEnabled:            true,
+		VPNHAEnabled:          len(seed.GetInfo().Spec.Provider.Zones) > 1,
```

Comment:
Please also add this in line 163, 183 and 203. You just fixed the default istio ingress gateway. However, the zonal gateways are affected as well because the highly available vpn is also used if a shoot uses failure tolerance `node`. Furthermore, the gateways for exposure classes are affected in the same way.

Reply:
On second thought, this might not even be correct. A seed can have highly available vpn-seed-server for control planes with fault tolerance `node` without even having multiple zones.
Did I get it wrong?
WDYT?

Reply:
Hm, yes very good point. With that, I think we need to add the labels in any case.